packages/block-library/src/image/edit.native.js

```diff
@@ -157,12 +163,18 @@ export default class ImageEdit extends React.Component {
 			</Toolbar>
 		);
 
-		const inlineToolbarButtons = (
-			<ToolbarButton
-				label={ __( 'Image Settings' ) }
-				icon="admin-generic"
-				onClick={ onImageSettingsButtonPressed }
-			/>
+		const getInspectorControls = () => (
+			<BottomSheet
+				isVisible={ this.state.showSettings }
+				title={ __( 'Image Settings' ) }
+				onClose={ onImageSettingsClose }
+				rightButtonConfig={ { text: __( 'Done' ), color: '#0087be', onPress: onImageSettingsClose } }
```

Comment:
Rather than a config object which format is hidden inside the `BottomSheet` component, we should use simple props such as `rightButtonText`, `rightButtonColor` and `righButtonPressed`.

An other alternative would be to expose Button from BottomSheet component and pass down an element instance instead:

```jsx
<BottomSheet
	isVisible={ this.state.showSettings }
	title={ __( 'Image Settings' ) }
	onClose={ onImageSettingsClose }
	rightButton={
		<BottomSheet.Button
			text={ __( 'Done' ) }
			color="#0087be"
			onPress={ onImageSettingsClose }
		/>
	}
>
...
</BottomSheet>